manuscript/manuscript.Rmd

```diff
@@ -491,68 +491,51 @@ Many of our most realistic ecological models exist only as numerical simulations
+And, at evaluation, if you give an agent a novel task, including tasks that are closely related to the examples that the agent saw in training, the agent can easily fail to generalize its learned behavior to this new situation [@pmlr-v97-cobbe19a].
+These failures are all due to the brittleness of deep neural networks. 
+Across deep learning, neural networks tend to perform exceptionally well on narrowly defined problems, yet neural networks show great fragility when given new situations or problem statements that require flexibility [@cogswell_reducing_2016].
+The ability to adapt to new situations is critical for any decision-making problem in the real world, so the current lack of this ability in deep RL may limit deep RL's effectiveness to conservation problems.
 
 <!--
 Under some conditions, RL agents can prove adept at _transfer learning_, in which an agent trained in one environment performs well under a different environment.
 -->
 <!-- if we're really recommending 'testing' for stability, maybe we need to demo that in the appendix-->
-
 <!-- reason 2: interpretability -->
-<!-- ML: Re our OU talk, might be worth saying that this is a barrier to adoption? If we want to ultimately use these RL agents to make important decisions, it would be hard to swallow us blindly using them without having any sense to what is going on. -->
-Interpretability is a common challenge at the adoption of most ML methods [@cite]. 
-Although there have been some attempts to increase the transparency of RL agents that use neural network as function approximators [cite], it is still very difficult to explain the behavior of RL-trained agents.
-As our examples have illustrated, it may be possible to gain intuition on an agent's behavior by visualizing the action selected against each possible state, though this becomes more difficult in large problems.
-Another challenge is that Deep RL algorithms are very sample inefficient, all though progress is being made to improve this [@cite]. <!-- ML: I think we can cut this sentence, mentionned it in passing earlier-->
--->
-
+Deep RL agents are also difficult to interpret [@heuillet_explainability_2021].
+Deep neural networks are commonly referred to as black box models, since it is challenging to fully explain why a deep neural network produces a specific output from a given input [@castelvecchi_can_2016].
+For deep RL, this means that we can rarely provide a meaningful explanation for why any agent selects an action.
+Compounded with the previously mentioned generalizability issues, the lack of explainability introduces a lot of safety concerns when attempting to apply these methods to the real world.
+When going from a simulator to the real world, the real world may introduce novel situations that a simulator-trained agent is not familiar with, and since we can not anticipate how an agent will respond to new situations, there is an inherent risk in letting RL agents make decisions in the real world. 
```

Comment:
I think this is basically a statement that any real-world use RL is really an exercise in transfer learning, since the real world isn't actually one of the environments it is trained on.  So the basic assumption is that interpretability might help intuition on when transfer learning will really transfer to the real world?

Reply:
Yes, like if the agent does something drastic, it'll be hard to put trust in the agent without having a sense of why it's making that action.

Aside: I'm sensing you mean "generalization" in place of transfer learning here. Transfer learning would more so be if you are transferring a policy to an agent that you'd then train in another environment. If you're training and agent then evaluating it on a different environment, then it's a generalization problem.